packaging/installer/UPDATE.md

```diff
@@ -46,81 +30,38 @@ The following table contains all possible installation types:
-you either used a non-standard install path, or that you don’t actually have Netdata installed).
+Otherwise, it should either indicate that the installation type is not supported (which probably means you either have a `custom` install or built Netdata manually) or indicate that it would create a new install (which means that you either used a non-standard install path, or that you don’t actually have Netdata installed).
 
-## Updates for most systems
+## UNIX
 
-In most cases, you can update netdata using our one-line installation script.  This script will automatically
-run the update script that was installed as part of the initial install (even if you disabled automatic updates)
-and preserve the existing install options you specified.
+In most cases, you can update Netdata using our one-line kickstart script. This script will automatically
+run the update script that was installed as part of the initial install and preserve the existing install options you specified.
 
-If you installed Netdata using an installation prefix, you will need to add an `--install-prefix` option specifying
-that prefix to this command to make sure it finds Netdata.
+If you installed Netdata using an installation prefix, you will need to add an `--install-prefix` option specifying that prefix to this command to make sure it finds Netdata.
 
 ```bash
 wget -O /tmp/netdata-kickstart.sh https://get.netdata.cloud/kickstart.sh && sh /tmp/netdata-kickstart.sh
 ```
 
-### Issues with older binpkg installs
-
-The above command is known not to work with binpkg type installs for stable releases with a version number of
-v1.33.1 or earlier, and nightly builds with a version number of v1.33.1-93 or earlier. If you have such a system,
-the above command will report that it found an existing install, and then issue a warning about not being able to
-find the updater script.
-
-On such installs, you can update Netdata using your distribution package manager.
-
-### Updates on hosts without IPv4 connectivity
-
-The update process outlined above suffers from the same issues that installing on hosts without IPv4
-connectivity does, and requires similar workarounds. For more details check [the explanation in our install
-documentation](/packaging/installer/README.md#installs-on-hosts-without-ipv4-connectivity).
-
-### If the kickstart script does not work
-
-If the above command fails, you can [reinstall
-Netdata](/packaging/installer/REINSTALL.md#one-line-installer-script-kickstartsh) to get the latest version. This
-also preserves your [configuration](/docs/netdata-agent/configuration/README.md) in `netdata.conf` or other files just like updating
-normally would, though you will need to specify any installation options you used originally again.
+## Windows
 
-## Docker
+To update a Windows Netdata installation, download the executable and proceed to reinstall the Agent. This procedure will update the installation.
```

Comment:
```suggestion
To update a Windows Netdata installation, download the executable and proceed to reinstall the Agent. This will update the installation.
```